From 84ca117864f4c6fc9c1e12295c22fb82f51b195a Mon Sep 17 00:00:00 2001
From: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Date: Thu, 20 Apr 2017 13:21:40 +0300
Subject: [PATCH 1716/5242] MLK-14548: video: dts: Remove unused regulator

commit  082e16b6e125587ff82be87e43cb6412adda67db from
https://source.codeaurora.org/external/imx/linux-imx.git

The reg_lcd_3v3 regulator is, theoretically, used by the display
controller eLCDIF as lcd supply.

When enabling the HDMI trasnmitter Sil902x, a gpio pin needed by the
regulator is taken by the HDMI TX as reset pin, so the regulator probe
fails and is never used.

The imx6sx-sdb-lcdif1.dts file is used for HDMI functionality, thus
enabling the HDMI transmitter invalidates any possible use of the
reg_lcd_3v3 regulator.

This patch removes the regulator enablement from the HDMI-specific dts.

Signed-off-by: Cristina Ciocan <cristina-mihaela.ciocan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
index 48e242e..2805404 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
@@ -9,12 +9,6 @@
 #include "imx6sx-sdb.dts"
 
 / {
-	regulators {
-		reg_lcd_3v3: regulator@4 {
-			status = "okay";
-		};
-	};
-
 	sii902x_reset: sii902x-reset {
 		status = "okay";
 	};
-- 
1.7.9.5

